thirdeye-server/src/main/java/ai/startree/thirdeye/service/AlertService.java

```diff
@@ -388,6 +390,11 @@ private void deleteAssociatedRcaInvestigations(final Long alertId) {
   }
 
   private long minimumLastTimestamp(final ThirdEyePrincipal principal, final AlertDTO dto) {
+    final String currentNamespace = authorizationManager.currentNamespace(principal);
```

Comment:
dto must have its namespace set when this method is called, so let's use 
```
dto.namespace()
```

Reply:
done